internal/codespaces/codespaces.go

```diff
@@ -79,3 +80,16 @@ func ConnectToLiveshare(ctx context.Context, progress progressIndicator, session
 		Logger:         sessionLogger,
 	})
 }
+
+// Helper function to connect to the internal RPC server and return an RPC invoker for it
+func CreateRPCInvoker(ctx context.Context, session *liveshare.Session, token string) (*rpc.Invoker, error) {
```

Comment:
Is there a reason to define this in the `codespaces` package? The convention would be to put it in the `rpc` package I think

Reply:
Moved to the `rpc` package. I was following what the `ConnectToLiveshare` function was doing but I agree that it doesn't make a whole log of sense.